app/src/main/java/org/eyeseetea/malariacare/layout/adapters/survey/navigation/NavigationController.java

```diff
@@ -223,6 +225,20 @@ private void visit(QuestionNode nextNode){
 
         //This node wont be shown in previous move (warnings)
         if(!nextNode.isVisibleInReview()){
+            //If the survey is sent we need hide the reminder question.
+            if(Session.getSurvey()!=null && !Session.getSurvey().isInProgress())
```

Comment:
@idelcano Add curly brackets for this 'if' please. In general at brackets because they make things more clear but special here that we have a loop inside  the if (not properly indented btw)